zebra/zebra_vty.c

```diff
@@ -995,6 +995,10 @@ static void vty_show_ip_route(struct vty *vty, struct route_node *rn,
 		if (CHECK_FLAG(re->status, ROUTE_ENTRY_QUEUED))
 			json_object_boolean_true_add(json_route, "queued");
 
+		if (CHECK_FLAG(re->status, ROUTE_ENTRY_ROUTE_REPLACING))
+			json_object_boolean_true_add(json_route,
```

Comment:
nit: `json_object_boolean_add(json_route, "routeReplacing", !!CHECK_FLAG(re->status, ROUTE_ENTRY_ROUTE_REPLACING))`

Reply:
none of the other booleans here are true/false like this.  If we want to fix( and I'm not necessarily convinced of that fact ) the whole thing should be fixed and that is a different problem and commit totally unrelated to this fix

Reply:
I gotta say I'm with Donatas here: let's start doing the correct/better thing with new code now (and look to fix old code eventually too.)

Reply:
I disagree it would show inconsistent output and be more trouble then it's worth

Reply:
as a further.  I intentionally choose only to add the flags to the route output if they are true on show json commands for the case where we have a full table.  Json output is already large enough lot's not make it worse here imo

Reply:
Can we change at least `Route Replacing` to `routeReplacing`?

Reply:
so ... I don't really get the problem here. hiding values (or hiding them _sometimes_, which seems even scarier) means that folks who are trying to use the json have to hard-code assumptions about what the default value is, about what absence of a key means - and that seems just ... wrong. let's just include the values, and make it easier on the folks who want to use the json output.

Reply:
In general I don't disagree, but some values are ok to hide and a flag being true is one of them imo, especially since I added these flags as a convenience for me to debug issues instead of having to manually decode `flags`.  If you want a standard about how json works then you should be submitting a change to the developer documentation instead of holding this code to some different unnamed standard.  Adding 5-10 lines of output per `show ip route json` is huge when there is a full feed.  We should be doing things to help our end users not slowing them down on stuff implemented to make our lives easier.  In any event I just remvoed the json code altogether.  I'll rely on the flags value in json from now on.